datasets/ecommerce/table.yaml

```diff
@@ -0,0 +1,29 @@
+charset: utf-8
```

Comment:
@qlinsey this is the table YAML, update-sample.csv is the datasets

Reply:
1

Reply:
Some message sent to me for this thread. Now cannot find from here, so sent to wrong person? thanks

Reply:
Maybe it is just a mistake. Let it go.

Reply:
ok. thanks